index.html

```diff
@@ -1,45 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Marke Parker</title>
-    <link rel="stylesheet" href="./assets/css/style.css">
-</head>
-<body>
-<!-- This a Header Section [Moussa Ibrahim]-->
-<main>
-    <header>
-        <nav class="header__nav">
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <link rel="stylesheet" href="./assets/css/font-awesome.css" />
+    <!-- fontawesome -->
 
-        </nav>
-    </header>
-</main>
-<!-- Read About My Life Section [SARA MAGDY]-->
-<section class="about section--py section"></section>
+  </head>
+  <body>
+    <!-- This a Header Section [Moussa Ibrahim]-->
+    <main>
+      <header>
+        <nav class="header__nav"></nav>
+      </header>
+    </main>
+    <!-- Read About My Life Section [SARA MAGDY]-->
+    <section class="about section--py section">
+      <div class="about__imgDiv">
```

Comment:
I really tried to use it but it didn't help me make the required look because the width was too little , I used justify-content: center instead of @AbdulrahmanMasoud

Reply:
ok, but Container must be used in all of the sections to Contain All Content
Like this
[IMG](https://drive.google.com/file/d/1ligMj5XdJtuRWoT3enorEj2A4L6GJcc_/view?usp=sharing)
if you use different width your section will be different from the other sections
"can you understand What I am meaning?"

Reply:
I understand that, but I tried and didn't bring the elements next to each other on the same line, and I didn't change anything in the width . 
where is the proplem if the result  look like the shape ?